### ISSUE TITLE
Installation Error on Windows 10 64bit

### ISSUE THREAD

**Issue by alexcruz0202** [peterq/pan-light#30]
Hi. 
I got following error on Windows 64bit.
"Get https://passport.baidu.com/v2/api/getqrcode?lp=pc: proxyconnect tcp: net/http: TLS handshake timeout". thanks

**Comment by peterq** [comment]
@alexcruz0202 
Did it come out when you click the button 'login with baidu qr code' ? it looks like you have a proxy setting in your system environment. please check if your setting is valid?

**Comment by alexcruz0202** [comment]
@peterq
You are correct. I have proxy setting in my system environment and it's no problem.
I know some apps independently needs to set proxy in configuration.
Regards.
